configs/application/components.json

```diff
@@ -106,18 +108,28 @@
 			},
 			"foreign": {
 				"services": {
-					"dockingService": {
-						"isArrangable": true
+					"windowService": {
+						"allowSnapping": true,
+						"allowGrouping": true,
+						"allowTabbing": true,
+						"allowTiling": true,
+						"allowAutoArrange": true,
+						"allowMinimize": true
+					},
+					"workspaceService": {
+						"persistURL": true
 					}
 				},
 				"components": {
 					"App Launcher": {
-						"launchableByUser": false
+						"launchableByUser": true
```

Comment:
Have you asked whether it is OK to put this component in the menu? It used to be available but was removed, so I am reluctant to re-add it with out confirmation.

Reply:
I plan to ask when creating the review & merge ticket for this PR, I have a very vague memory that they were removed due to issues with the titlebar long ago (probably around the time I joined). I added them in to test them - but once done they seemed a reasonable addition.

Reply:
Those should be set to false. We don't want to launch "getting started tutorial" or "finsemble documentation" from the app menu. I'll push up the change.